docs/content/docs/test/Runner.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Test Runner
+type: docs
+---
+
+# Avito test runner
+
+## How to run tests in kubernetes from a local machine (internal)
+
+1. Get access to kubernetes cloud: [internal doc](http://links.k.avito.ru/Kubectl)
+1. [Request](http://links.k.avito.ru/androidEmulatorServiceDesk) `exec` access to `android-emulator` namespace in `beta` cluster
+1. Setup a context on `beta/android-emulator` with your user access (also described in first doc)
+1. Run tests with extra parameters specified, example for `:test-app`:
+
+```shell script
+./gradlew :subprojects:android-test:test-app:instrumentationUi 
+    -Pci=true 
+    -PkubernetesContext=<your context>
```

Comment:
It's not clear what context I need to type here.

Reply:
it is described in list above

Reply:
and you need to know about https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#define-clusters-users-and-contexts
beforehand